kolibri/plugins/coach/assets/src/views/home/HomePage/LessonsBlock.vue

```diff
@@ -48,6 +47,62 @@
     $trs: {
       viewAll: 'All lessons',
     },
+    computed: {
+      ...mapState('classSummary', ['groupMap']),
+      ...mapGetters('classSummary', [
+        'learners',
+        'lessons',
+        'lessonLearnerStatusMap',
+        'getLessonStatusCounts',
+      ]),
+      table() {
+        const recent = sortBy(this.lessons, this.lastActivity).slice(0, MAX_LESSONS);
```

Comment:
The `lastActivity` method is returning the right dates, but `sortBy` isn't sorting them correctly..

Reply:
Use this instead

```
        const recent = orderBy(this.exams, this.lastActivity, ['desc']);

```